.github/workflows/testing.yml

```
@@ -401,4 +376,4 @@ jobs:
           files: ./coverage.xml
           flags: win-longtests
           name: codecov-umbrella
-          fail_ci_if_error: false
+          fail_ci_if_error: false
```

```suggestion
          fail_ci_if_error: false

```

Adding blank line at end of file for safety.  github yml can be notoriously picky.